packages/core/src/node/test/cluster-test-worker.ts

```diff
@@ -15,9 +15,9 @@ const jobs: { [id: string]: (() => Promise<void>) | undefined } = {
         try {
             await server.restart();
             firstRestartFailed = false;
-        } catch (e) {
-            if ((e as Error).message.indexOf('failed to restart') === -1) {
-                throw e;
+        } catch (error) {
+            if (!/failed to (re)?start/.test(error.message)) {
```

Comment:
`(re)?` - redundant?

Reply:
it is one of the cases where I would really love to have something along https://github.com/theia-ide/theia/pull/1536